### ISSUE TITLE
java.lang.String.indent is not in Java 11

### ISSUE THREAD

**Issue by magnus-madsen** [flix/flix#1083]
The `java.lang.String.indent` method is not available in Java 11 -- which unfortunately breaks one of our build servers. 

```
>> Undefined object method 'indent' in class 'java.lang.String.

872 |>         import java.lang.String.indent(Int32);
873 |>         let s1 = s.indent(n) as & Pure;
874 |>         replaceMatches("\r?\n", lineSeparator(), s1)

undefined method.

No method matches the signature:
  indent(int)
```

In the short term I am either going to remove this specific method `String.indent` or request a rewrite that does not depend on `java.lang.String.indent`. 

What do you think Stephen?

**Comment by magnus-madsen** [comment]
By the way: We should probably have a separate discussion about what Java versions we intend to target. For now, I just want to get the build server running again :)

**Comment by stephentetley** [comment]
Hi Magnus, I'll look at writing an "in Flix"  replacement this evening.

**Comment by stephentetley** [comment]
I've sent a pull request with in Flix versions of `indent` and `stripIndent`:

String indent "in Flix" #1090

**Comment by magnus-madsen** [comment]
Fixed in #1090
